example/src/components/ComponentDocTemplate.js

```diff
@@ -159,36 +174,36 @@ function ComopnentDocTemplate(props) {
         dummyElement.select()
         document.execCommand("copy")
         document.body.removeChild(dummyElement)
-        // let confirmCopy = document.getElementById('confirmCopy')
-        // confirmCopy.style.display = 'flex'
+
+        copiedConfirmation.style.display = 'flex'
     }
 
-    const { componentName, componentDescription, exampleText, templateProps } = props.templateInfo
-
-    let newArr = templateProps.map((availableProp, idx) => {
-        const { propName, propType, propDescription, options } = availableProp
-        let optionListString = options.map(option => {
-            return `${option}, `
-        })
-
-        return (
-            <PropsInfoContainer key={idx}>
-                <InfoWrapper>
-                    <InfoBold>prop:</InfoBold>
-                    <InfoRegular>{propName}</InfoRegular>
-                </InfoWrapper>
-                <InfoWrapper>
-                    <InfoBold>type:</InfoBold>
-                    <InfoRegular>{propType}</InfoRegular>
-                </InfoWrapper>
-                <InfoWrapper>
-                    <InfoBold>options:</InfoBold>
-                    <InfoRegular>{optionListString}</InfoRegular>
-                </InfoWrapper>
-                <InfoDescription>{propDescription}</InfoDescription>
-            </PropsInfoContainer>
-        )
-    })
+    const { componentName, componentDescription } = props.templateInfo
+    // templateProps & exampleText
+
+    // let newArr = templateProps.map((availableProp, idx) => {
+    //     const { propName, propType, propDescription, options } = availableProp
+    //     let optionListString = options.map(option => {
+    //         return `${option}, `
+    //     })
+    //     return (
+    //         <PropsInfoContainer key={idx}>
+    //             <InfoWrapper>
+    //                 <InfoBold>prop:</InfoBold>
+    //                 <InfoRegular>{propName}</InfoRegular>
+    //             </InfoWrapper>
+    //             <InfoWrapper>
+    //                 <InfoBold>type:</InfoBold>
+    //                 <InfoRegular>{propType}</InfoRegular>
+    //             </InfoWrapper>
+    //             <InfoWrapper>
+    //                 <InfoBold>options:</InfoBold>
+    //                 <InfoRegular>{optionListString}</InfoRegular>
+    //             </InfoWrapper>
+    //             <InfoDescription>{propDescription}</InfoDescription>
+    //         </PropsInfoContainer>
+    //     )
+    // })
```

Comment:
Delete commented code if we're done with it (we can always look back on past commits if we need to see what things looked like for reference)